git_machete/client.py

```diff
@@ -1851,7 +1852,11 @@ def checkout_github_prs(self,
         remote, (org, repo) = self.__derive_remote_and_github_org_and_repo()
         current_user: Optional[str] = git_machete.github.derive_current_user_login()
         if not current_user and my_opened_prs:
-            warn("Could not determine current user name, please check your token.")
+            if called_function:
+                warn("Could not determine current user name, please check your token.")
+                return
+            else:
+                raise MacheteException("Could not determine current user name, please check your token.")
         all_open_prs: List[GitHubPullRequest] = derive_pull_requests(org, repo)
         valid_prs: List[GitHubPullRequest] = self.__get_valid_pull_requests(pr_nos,
```

Comment:
I'd rather rename it to `__get_applicable_pull_requests`, same with the var name

Reply:
or `qualifying`?

Reply:
Ok, `applicable` makes more sense to me, will change it.